modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs

```diff
@@ -99,6 +99,28 @@ public void TestGlobalMetrics()
             Assert.AreEqual(1, remoteMetrics.CachePuts);
         }
 
+        /// <summary>
+        /// Tests the cache metrics enable/disable 
+        /// </summary>
+        [Test]
+        public void TestCacheEnableStatistics()
+        {
+            TestEnableStatistics("cacheEnableStatistics", (cache, b) => cache.EnableStatistics(b));
+        }
+
+        
+        /// <summary>
+        /// Tests the cache metrics enable/disable 
+        /// </summary>
+        [Test]
+        public void TestClusterGroupEnableStatistics()
+        {
+            var cacheName = "clusterEnableStatistics";
+            TestEnableStatistics(
+                cacheName,
+                (cache, b) => cache.Ignite.GetCluster().EnableStatistics(new[] {cacheName}, b));
```

Comment:
Please add the following tests additionally:
* Pass empty `cacheNames` 
* Pass non-existent cache names